e2e/components/CreateFlows/CreateFullPage-test.avt.e2e.js

```diff
@@ -25,4 +27,29 @@ test.describe('CreateFullPage @avt', () => {
       'CreateFullPage @avt-default-state'
     );
   });
+
+  test('@avt-error-disabled-state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'CreateFullPage',
+      id: 'ibm-products-patterns-create-flows-createfullpage--create-full-page',
+      globals: {
+        carbonTheme: 'white',
+      },
+    });
+    const inputElement = await page.locator('#test-1');
+    await inputElement.click();
+    await page.mouse.click(0, 0);
+    const dataInvalid = await inputElement.getAttribute('data-invalid');
+    const nextButtonElement = page.locator('button:has-text("Next")');
+    const backButtonElement = page.locator('button:has-text("Back")');
+    const isNextDisabled = await nextButtonElement.isDisabled();
+    const isBackDisabled = await backButtonElement.isDisabled();
```

Comment:
Hey, I think by default the input box is will be empty always and dataInvalid will be true.
After checking the invalid state and disabled state, can we add some text to the input filed to test  valid state and next button enabled state as-well.